ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/recovery/arjunacore/XARecoveryModule.java

```diff
@@ -1136,6 +1149,7 @@ private enum ScanStates {
 
     private final List<XAResourceOrphanFilter> _xaResourceOrphanFilters;
 
+    // Once JBTM-3953 is completed, convert this to normal boolean
```

Comment:
If you haven't already, I would recommend you to raise a JBTM for this work. Is that OK?

Reply:
Yep, it should be [this one](https://issues.redhat.com/browse/JBTM-3953)

Reply:
Well, this modification should be part of JBTM-3953

Reply:
Thanks, I was thinking specifically about the conversion to the normal boolean?